sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob/BlobContainerAsyncClientJavaDocCodeSnippets.java

```diff
@@ -440,6 +440,45 @@ public void generateSasWithContext() {
         // END: com.azure.storage.blob.BlobContainerAsyncClient.generateUserDelegationSas#BlobServiceSasSignatureValues-UserDelegationKey-String-Context
     }
 
+    /**
+     * Code snippet for {@link BlobContainerAsyncClient#createIfNotExists()} and
+     * {@link BlobContainerAsyncClient#createIfNotExistsWithResponse(Map, PublicAccessType)}
+     */
+    public void createIfNotExistsCodeSnippets() {
+        // BEGIN: com.azure.storage.blob.BlobContainerAsyncClient.createIfNotExists
+        client.createIfNotExists().subscribe(
+            response -> System.out.printf("Create completed%n"),
+            error -> System.out.printf("Error while creating container %s%n", error));
```

Comment:
Let's add another line to these samples along the lines of
```
.switchIfEmpty(Mono.<String>empty().doOnTerminate(()-> System.out.println("Already existed")))
```
that will also give the customers an example of how to handle the case where the object already existed in case they have a scenario for that.